lib/get-html.ts

```diff
@@ -29,7 +29,14 @@ export const getHtml = async (
 
   // get only the HTML we care about to avoid unnecessary cache updates
   $('[data-testid="callout"]').remove();
-  const html = $("#article-contents").parent().parent().html() ?? "";
+  const data = $("#article-contents").parent().parent();
+  data.find("*").each((i, el) => {
+    const classes = $(el).attr("class");
+    if (classes && !classes.includes("language-json") && !classes.includes("warning")) {
```

Comment:
Might as well default to an empty array to save a check (|| save using optional chain instead of the check)

```suggestion
    const classes = $(el).attr("class") ?? [];
    if (!classes.includes("language-json") && !classes.includes("warning")) {
```

Reply:
huh weird that it makes typescript unhappy - @wolfy1339 feel free to just go with what you originally had 🤷